front/src/Phaser/Game/GameScene.ts

```diff
@@ -614,6 +616,15 @@ export class GameScene extends DirtyScene {
             this.openChatIcon.setVisible(!v);
         });
 
+        this.emoteUnsubscribe = emoteStore.subscribe(() => {
```

Comment:
When you subscribe, the first parameter of the callback is the new value. So you can write:
```
.subscribe((emoteKey) => {
            if (emoteKey) {
                ...
                }
                });